src/plugins/foxPage/components/FoxChart.tsx

```diff
@@ -0,0 +1,74 @@
+import { Box, StatArrow, StatNumber, Text } from '@chakra-ui/react'
+import { HistoryTimeframe } from '@shapeshiftoss/types'
+import { useState } from 'react'
+import NumberFormat from 'react-number-format'
+import { useTranslate } from 'react-polyglot'
+import { Card } from 'components/Card/Card'
+import { TimeControls } from 'components/Graph/TimeControls'
+import { PriceChart } from 'components/PriceChart/PriceChart'
+import { RawText } from 'components/Text/Text'
+import { useLocaleFormatter } from 'hooks/useLocaleFormatter/useLocaleFormatter'
+import { selectMarketDataById } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+type FoxChartProps = {
+  assetId: string
+}
+
+export const FoxChart: React.FC<FoxChartProps> = ({ assetId }) => {
+  const [timeframe, setTimeframe] = useState(HistoryTimeframe.MONTH)
```

Comment:
Note: We'll want to use `DEFAULT_HISTORY_TIMEFRAME` when https://github.com/shapeshift/web/pull/1859/ is merged